backend/src/zelthy/apps/auditlogs/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Lets have a readme file in this folder where we detail out:
- which version of auditlog library we have taken
- what changes have we made
- overall rationale
- How is it being used, how to exclude model, fields, etc.
Also include a section in Docs